rpcutils/rpcclient.go

```diff
@@ -220,3 +220,34 @@ func GetBlock(ind int64, client *rpcclient.Client) (*dcrutil.Block, *chainhash.H
 
 	return block, blockhash, nil
 }
+
+// GetTransaction get a transaction with a given hash
```

Comment:
// GetTransactionVerboseByID...

Reply:
but this is a confusing name since it's taking a hash.  Let's call it `GetTransactionVerbose` unless that already exists